cmake/dalienv.yml.in

```diff
@@ -2,6 +2,7 @@ name: dalienv
 dependencies:
   - python=3.8
   - pip
+  - numpy
```

Comment:
I'm sorry to say that it is not a good change.
First of all, DALI doesn't depend on the numpy, you can just build a flow that uses numpy as an auxiliary tool but it is not a must.
This way we would need to add a dependency to Torch and CuPy as DALI can use tensors from these frameworks.

Reply:
I added a specific use case in the linked issue, could you recommend how to solve it without numpy?

Reply:
In this case I believe you need numpy to create the serialized pipeline you would later use in during the inference through the DALI backend. During the inference process itself, the numpy package is not needed, so you can use any environment with numpy, not necessarily inside the TRITON dev image.